src/kernels/MIOpenBatchNormFwdTrainSpatial.cl

```diff
@@ -145,8 +145,8 @@ MIOpenBatchNormFwdTrainSpatial(const __global _FLOAT* __restrict in,
         variance = 0;
     }
     invVariance = rsqrt(variance + (_FLOAT_ACCUM)epsilon);
-    pvscale     = (_FLOAT_ACCUM)lcl_scale;
-    pvbias      = (_FLOAT_ACCUM)lcl_bias;
+    pvscale     = FLOAT2ACCUM(lcl_scale);
+    pvbias      = FLOAT2ACCUM(lcl_bias);
```

Comment:
this seems off.  lcl_scale and lcl_bias are actually _FLOAT_PREC, not FLOAT.  so technically a FLOATPREC2ACCUM?

Reply:
Oh in that case we don't need the change.

Nice catch!